recipes/scicookie/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "scicookie" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scicookie-{{ version }}.tar.gz
+  sha256: b1f6c94b9370a8ef21e5591ff091dc931a0dec806bca7245a5741739c190a214
+
+build:
+  skip: true  # [py>=400 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }} 
+  host:
+    - python 
+    - poetry-core >=1.0.0
+    - pybind11
+    - pip
+  run:
+    - python 
+    - cookieninja 1.0.0
+    - sh >=2.0.4,<3.0.0
+    - colorama >=0.4.6,<0.5.0
+    - inquirer >=3.1.3,<4.0.0
+    - pyyaml >=6.0.1
+
+test:
+  imports:
+    - scicookie
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://example.com/scicookie-home
+  summary: Cookiecutter template for a Python package
+  license: Apache-2.0 AND BSD-3-Clause
+  license_file:
+    - LICENSE
+    - src/scicookie/{{ cookiecutter.project_slug }}/LICENSE
+
+extra:
+  recipe-maintainers:
+    - ayeankit
```

```suggestion
    - ayeankit
    - xmnlab
```

please add me as a maintainer :)